README.md

```diff
@@ -39,6 +39,8 @@ pip install mcp-server-git
 python -m mcp_server_git
 ```
 
+Follow [these](https://docs.astral.sh/uv/getting-started/installation/) instructions to install `uv` / `uvx` and [these](https://pip.pypa.io/en/stable/installation/) to install `pip`.
```

Comment:
`pip` should be installed with ~any Python distribution, so we can just focus on uv:
```suggestion
Follow [these](https://docs.astral.sh/uv/getting-started/installation/) instructions to install `uv` / `uvx`.
```